Fixed typo on line 38 (#1162)
vuejs/vue-devtools: README.md

### Manual Installation

> This is only necessary when you want to build the extension by yourself from source to get not-yet-released features.

Reviewer: ```suggestion
This is only necessary when you want to build the extension with the source repo to get not-yet-released features.
```

Reviewer: docs: improving phrasing
